app/ide-desktop/lib/client/src/index.ts

```diff
@@ -130,8 +129,10 @@ class App {
                 chromeOptions.push(chromeOption)
             }
         }
-        const add = (option: string, value?: string) =>
+        /** Adds the specified Chrome option. */
+        function add(option: string, value?: string) {
             chromeOptions.push(new configParser.ChromeOption(option, value))
+        }
```

Comment:
why we need to change const lambdas int functions? This makes the code more verbose.